### ISSUE TITLE
Create the system user when upgrading from 2.6.0 to 2.7.0

### ISSUE THREAD

**Issue by dwaynebailey** [translate/pootle#3726]
I get the following error when trying to view the timeline on this unit:
http://linode2.locamotion.org/ach/firefox-beta/translate/#filter=all&unit=17545118

This is a post mlo migration, so it could be related to bad migration.  But I guess we either have users or the code should handle the case where a user was deleted.

```pytb
Environment:


Request Method: GET
Request URL: http://linode2.locamotion.org/xhr/units/17545118/timeline/

Django Version: 1.7.8
Python Version: 2.7.6
Installed Applications:
['django.contrib.sessions',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.humanize',
 'django.contrib.sites',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'accounts',
 'contact',
 'import_export',
 'pootle',
 'pootle_app',
 'pootle_misc',
 'pootle_store',
 'pootle_language',
 'pootle_project',
 'pootle_translationproject',
 'pootle_profile',
 'pootle_statistics',
 'pootle_notifications',
 'pootle_autonotices',
 'pootle_terminology',
 'staticpages',
 'virtualfolder',
 'django_assets',
 'django_rq',
 'overextends',
 'evernote_auth',
 'evernote_reports',
 'django_extensions']
Installed Middleware:
['pootle.middleware.baseurl.BaseUrlMiddleware',
 'django.middleware.cache.UpdateCacheMiddleware',
 'pootle.middleware.cache.CacheAnonymousOnly',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.middleware.http.ConditionalGetMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'pootle.middleware.setlocale.SetLocale',
 'django.contrib.messages.middleware.MessageMiddleware',
 'pootle.middleware.errorpages.ErrorPagesMiddleware',
 'django.middleware.common.CommonMiddleware',
 'pootle.middleware.captcha.CaptchaMiddleware',
 'django.middleware.cache.FetchFromCacheMiddleware']


Traceback:
File "/var/www/sites/pootle-mlo2/env/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  111.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/var/www/sites/pootle-mlo2/env/local/lib/python2.7/site-packages/newrelic-2.50.0.39/newrelic/packages/wrapt/wrappers.py" in __call__
  517.                 args, kwargs)
File "/var/www/sites/pootle-mlo2/env/local/lib/python2.7/site-packages/newrelic-2.50.0.39/newrelic/hooks/framework_django.py" in wrapper
  499.                 return wrapped(*args, **kwargs)
File "/var/www/sites/pootle-mlo2/env/local/lib/python2.7/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  52.         response = view_func(request, *args, **kwargs)
File "/var/www/sites/pootle-mlo2/src/pootle/pootle/apps/pootle_store/decorators.py" in decorated_f
  76.             return f(request, unit, *args, **kwargs)
File "/var/www/sites/pootle-mlo2/src/pootle/pootle/apps/pootle_store/views.py" in timeline
  625.             'submitter': User.objects.get_system_user(),
File "/var/www/sites/pootle-mlo2/src/pootle/pootle/apps/accounts/managers.py" in get_system_user
  69.         return super(UserManager, self).get_queryset().get(username='system')
File "/var/www/sites/pootle-mlo2/env/local/lib/python2.7/site-packages/django/db/models/query.py" in get
  357.                 self.model._meta.object_name)

Exception Type: DoesNotExist at /xhr/units/17545118/timeline/
Exception Value: User matching query does not exist.
```

**Comment by julen** [comment]
The *system* user is as integral as the *nobody* and *default* users — all command-line actions are attributed to this user. Bad migration, hence.

**Comment by dwaynebailey** [comment]
This missing migration is in cbd26d8b7f57afcf299535bc7beb87cd07038a51

**Comment by unho** [comment]
Merged a fix in 01d7ceb7, so closing this.
